pkg/storaged/block/format-dialog.jsx

```diff
@@ -185,6 +185,12 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     add_fsys("ext4", { value: "ext4", title: "EXT4" });
     add_fsys("vfat", { value: "vfat", title: "VFAT" });
     add_fsys("ntfs", { value: "ntfs", title: "NTFS" });
+    if (client.in_anaconda_mode()) {
+        if (block_ptable && block_ptable.Type == "gpt")
+            add_fsys(true, { value: "biosboot", title: "BIOS boot partition" });
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19826-20240109-133522-cb1438ac-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)